scripts/prepare_transport_data.py

```diff
@@ -138,23 +138,23 @@ def prepare_transport_data(n):
 
     # divide out the heating/cooling demand from ICE totals
     # and multiply back in the heating/cooling demand for EVs
-    ice_correction = (transport_shape * (1 + dd_ICE)).sum() / transport_shape.sum()
-
-    if snakemake.config["custom_data"]["transport_demand"]:
-        energy_totals_transport = nodal_energy_totals["total road"]
-
-        transport = transport_shape.multiply(energy_totals_transport) * 1e6 * Nyears
-    else:
-        energy_totals_transport = (
-            nodal_energy_totals["total road"]
-            + nodal_energy_totals["total rail"]
-            - nodal_energy_totals["electricity rail"]
-        )
-        transport = (
-            (transport_shape.multiply(energy_totals_transport) * 1e6 * Nyears)
-            .divide(efficiency_gain * ice_correction)
-            .multiply(1 + dd_EV)
-        )
+    ice_correction = (transport_shape * (1 + dd_ICE)).sum() / transport_shape.sum() # currently not used same for dd_EV
+
+    # if snakemake.config["custom_data"]["transport_demand"]:
+    energy_totals_transport = nodal_energy_totals["total road"]
```

Comment:
Don't we have to consider the rail transport as well?

Reply:
Now it is added in prepare_sector_network, the function name is add_rail_transport

Reply:
Oh ok, saw it now!